user/urls.py

```diff
@@ -2,14 +2,17 @@
 from django.urls import path, include
 from django.views.generic import TemplateView
 from django.contrib.auth.views import LogoutView
-from .views import shows,add_movie,movie_list,AddShowView
+from .views import shows, add_movie, movie_list, AddShowView,MovieAutocomplete, redirect_to_shows
+
 urlpatterns = [
-    
     path('admin/', admin.site.urls),
-    path('shows/', shows, name='shows'),
-    path('add_movie/', add_movie, name='add_movie'),
-    path('movie_list/', movie_list, name='movie_list'),
-    path('add_show/', AddShowView.as_view(), name='AddShowView'),
-    
-    
-]
\ No newline at end of file
+
+    # Redirect root URL to /shows/
+     path('', redirect_to_shows, name='redirect_to_shows'),
+
+    path('user/shows/', shows, name='shows'),
```

Comment:
why have you added an extra user?
This means the new url will be http://127.0.0.1:8000/user/user/shows/